src/renderer/index.ts

```diff
@@ -2,5 +2,10 @@ import * as replugged from "./replugged";
 
 window.replugged = replugged;
 
-await replugged.plugins.loadAll();
-await replugged.ignition.ignite();
+// Splash screen
+if (document.title === "Discord Updater") {
```

Comment:
I don't think this will work with non-english languages. Is there a better way you can determine this?

Reply:
Will do, good catch

Reply:
Ended up not being needed, although I found a better way anyway.